### ISSUE TITLE
A way to fix add tokens and a 999,999,999 IQ Suggestion

### ISSUE THREAD

**Issue by SalamiHecker** [glixzzy/blooket-hack#1179]
Hey everyone and @glixzzy.

I'm sure that you know that the 'add tokens' hack only gives 500 token every day. I think that if you are able to make a 'set time until the token limit resets', then you could repeat the process for the 500 tokens. It should be easy for you since you know how to set weight on Fishing Frenzy.

If you managed to read this suggestion, I wanted to say that I have now overthrown IT at my school and I admire you very much.

**Comment by BlooketHackerx** [comment]
genius!!

**Comment by Briggan311** [comment]
ya and it is a good way to get banned from blooket

**Comment by derinpilge** [comment]
Good idea. If you contact basil from gbasil.dev/blooket I think he can make one for you in a few days or 2 weeks

**Comment by Briggan311** [comment]
20$ bets it is a scam

**Comment by derinpilge** [comment]
> 20$ bets it is a scam

no its not i use it since school cheats is blocked by iboss cloud security

**Comment by SalamiHecker** [comment]
> ya and it is a good way to get banned from blooket

@Briggan311 I can make another account anytime and I make my own hacks to **SET THE TIME UNIL MY ACCOUNT IS UNBANNED**

**Comment by SalamiHecker** [comment]
> > Not possible, it is serversides
> 
> @coolman3289 isn't 'get all blooks' serverside?

### Blooket has it's own timer that starts the first time you get tokens that day. 
If you can change your weight on 'Fishing Frenzy' then you can change the timer too.

I don't like to have these arguments. If you argue more I will not respond. **For now, if you have the time to argue, then you have time to do these:**

- [ ] Find a way to tinker with 'setWeight.js' ([https://github.com/glixzzy/blooket-hack/blob/main/fishing-frenzy/setWeight.js](url)) so that you get 'setTimeBeforeReset.js (in a new place of coarse... Don't change the actual code in this respitory)
- [ ] Don't think you know everything (no one knows everything) and that it is still worth a try

**Comment by coolman3289** [comment]
It’s SERVERSIDED, you can’t do it

**Comment by gBasil** [comment]
> Good idea. If you contact basil from gbasil.dev/blooket I think he can make one for you in a few days or 2 weeks

How about I attach you to the body of a rocket and launch you into low-Earth orbit.
